frontend/src/app/restaurant-suggestions-list/restaurant-suggestions-list.component.spec.ts

```diff
@@ -0,0 +1,28 @@
+import {async, ComponentFixture, TestBed} from '@angular/core/testing';
+
+import {RestaurantSuggestionsListComponent} from './restaurant-suggestions-list.component';
+import {RestaurantComponent} from "../restaurant/restaurant.component";
+import {MatCardModule} from "@angular/material";
+
+describe('RestaurantSuggestionsListComponent', () => {
+  let component: RestaurantSuggestionsListComponent;
+  let fixture: ComponentFixture<RestaurantSuggestionsListComponent>;
+
+  beforeEach(async(() => {
+    TestBed.configureTestingModule({
+      imports: [MatCardModule],
+      declarations: [RestaurantSuggestionsListComponent, RestaurantComponent]
+    })
+      .compileComponents();
+  }));
+
+  beforeEach(() => {
+    fixture = TestBed.createComponent(RestaurantSuggestionsListComponent);
+    component = fixture.componentInstance;
+    fixture.detectChanges();
+  });
+
+  it('should create', () => {
+    expect(component).toBeTruthy();
+  });
```

Comment:
You should at least test that your restaurants are displayed

Reply:
Yeah of course . Sorry .
I didn't do it with tdd 🤣